schema/xsd/direct.xsd

```diff
@@ -59,6 +59,7 @@
                 </xs:element>
             </xs:choice>
             <xs:element name="signature-type" type="signature-type" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="on-behalf-of" type="signing-on-behalf-of" minOccurs="0" maxOccurs="1"/>
```

Comment:
Er nok bare jeg som ikke er helt inni dette her, men hvorfor har vi "dynamisk defaultverdi" i portaloppdrag?

Ellers enig i at vi bør gjøre det likt for redirect og portal, d.v.s. ha standardverdien definert i koden.